cli.js

```diff
@@ -134,13 +147,57 @@ async function init() {
 
 	ee.on('finish', async () => {
 		try {
+			const hasRaw = fs.existsSync(rawSlaFile);
+			const hasRtf = fs.existsSync(rtfSlaFile);
+			const hasTxt = fs.existsSync(txtSlaFile);
+
+			if (hasRaw || hasRtf || hasTxt) {
+				ora.text = 'Adding Software License Agreement';
+
+				const tempDmgPath = tempy.file({extension: 'dmg'});
+				// UDCO or UDRO format is required for SLA
+				await execa('/usr/bin/hdiutil', ['convert', '-format', 'UDCO', dmgPath, '-o', tempDmgPath]);
+				await execa('/usr/bin/hdiutil', ['unflatten', tempDmgPath]);
+
+				if (hasRaw) {
+					await execa('/usr/bin/rez', ['-a', rawSlaFile, '-o', tempDmgPath]);
+				} else {
+					let data = fs.readFileSync(path.join(__dirname, 'base.r'), 'utf8');
+					let plainText = '';
+
+					data += '\ndata \'RTF \' (5000, "English") {\n';
+
+					if (hasRtf) {
+						data += serializeString((Buffer.from(fs.readFileSync(rtfSlaFile, 'utf8')).toString('hex').toUpperCase()));
```

Comment:
> Buffer.from(fs.readFileSync(rtfSlaFile, 'utf8'))

If you drop the `utf8` the file will be read directly as a buffer.

Reply:
Done